test/grpc/cancelled-by-server.cc

```diff
@@ -23,7 +23,7 @@ using eventuals::grpc::CompletionPool;
 using eventuals::grpc::Server;
 using eventuals::grpc::ServerBuilder;
 
-TEST_F(EventualsGrpcTest, CancelledByServer) {
+TEST(CancelledTest, CancelledByServer) {
```

Comment:
Ditto on staying consistent with naming based on files, suggestion:
```suggestion
TEST(CancelledByServerTest, Cancelled) {
```

Reply:
Done